### PR TITLE
gdscript support

### PR DESCRIPTION
**Pull request by EmmChriss**

### CODE REVIEW

File: languages.toml [hunk 1]
```diff
@@ -1038,3 +1038,21 @@ roots = []
...
+injection-regex = "gdscript"
+file-types = ["gd"]
+shebangs = []
+roots = ["project.godot"]
+auto-format = true
+comment-token = "#"
+indent = { tab-width = 4, unit = "    " }
+grammar = "gdscript"
+# TODO: change this when socket support is added
+language-server = { command = "nc", args = ["localhost", "6008"] }
```
Comment (the-mikedavis) [inline languages.toml:1054]: This is an interesting hack. I'm a little worried it won't be cross-platform: does windows have netcat?

I think until socket support is added, it makes sense to remove the `language-server` configuration here and configure it in your `~/.config/helix/languages.toml`, maybe with some notes in the new [lsp wiki](https://github.com/helix-editor/helix/wiki/How-to-install-the-default-language-servers). What do you think?

Comment (EmmChriss) [reply languages.toml:1054]: That sounds good.
Maybe I could write a python hack that's a bit more complicated but cross-platform.
Or a binary could be included with helix for this.
But until then, I agree, this should be removed.

File: runtime/queries/gdscript/indents.toml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+indent = [
```
[the remark this answers is not in the document]
Comment (the-mikedavis) [reply runtime/queries/gdscript/indents.toml:1]: Yep the release docs now live separately from the master docs. For indent docs, these are now current: https://docs.helix-editor.com/master/guides/indent.html

**Comment by archseer** [comment]
Can you run `cargo xtask docgen`?

**Pull request merged**